lib/src/mail/mail_client.dart

```diff
@@ -830,16 +835,19 @@ class MailClient {
   ///
   /// Optionally specify the [sentMailbox] when the mail system does not
   /// support mailbox flags.
+  /// first
   Future<void> sendMessage(
     MimeMessage message, {
     MailAddress? from,
     bool appendToSent = true,
+    bool supportUnicode = false,
```

Comment:
should this be renamed to `useUnicodeSenderAddress` perhaps instead (not sure if I got this right)? Also, is there a way to determine this automatically without requiring the user to specify this manually?

Reply:
you are right